k8s.io/deployment.yaml

```diff
@@ -57,8 +61,12 @@ spec:
           failureThreshold: 2
         lifecycle:
           preStop:
-            exec: # nginx likes to terminate ASAP
-              command: [ "/bin/sleep", "30" ]
+            exec:
+              command:
```

Comment:
is this already running or is it a change?

Reply:
A change not tested in the canary namespace.